addons/api/mirage/scenarios/ipc.js

```diff
@@ -199,7 +199,7 @@ export default function initializeMockIPC(server, config) {
      * Check for OS chrome state
      */
     showWindowActions() {
-      return true;
+      return window.navigator?.userAgentData?.platform === 'Windows';
```

Comment:
should these window actions also be shown on linux?

Reply:
Good call! building a VM on linux to validate 😉

Reply:
Which scenarios is `userAgentData` available? I don't see it firefox or safari (is this chromium only?) but I do see it in `navigator.userAgent` though it would require you to pull it out. If we go down this path, should we use that one instead or as a backup field? And default to true if we don't find the properties?

Reply:
`userAgenData` is indeed not present on firefox, and for the moment available in chromium. [More info about it here](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/userAgentData). I assume, since this is DC that runs on Electron which uses chrome it was ok to use, also because the `userAgent.platform` [its been deprecated as per this information](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/platform).

I will look for a more compatible solution, open to suggestions 😉